docs/pages/actions/monday/api-reference/create-items.mdx

```diff
@@ -0,0 +1,24 @@
+---
+openapi: "POST /api/actions/monday/items/create"
+---
+
+<RequestExample>
+
+```bash Request
+curl --request POST \
+  --url https://app.vessel.dev/api/actions/monday/items/create \
+  --header 'Content-Type: application/json' \
+  --header 'x-vessel-api-token: ' \
+  --header 'x-vessel-access-token: '\
+  --data '{
+    "boardId": "123456789",
```

Comment:
Ah these should be raw numbers, without the quotes

Reply:
ah whoops